### DISCUSSION TITLE
Installed it, gallery created... no lightbox?

### DISCUSSION THREAD

**Discussion by JuliusFroglet** [archetyped/simple-lightbox#1013] [Q&A] [answered]
Hi, I just installed the simple Lightbox Plugin,
I have my Gallerys in my sites created by default wordpress. They are all set to Mediafile.
However, the Lightbox does not appear.

Any Ideas? 
Do I have to create the Imagelinks manually?

**Comment by archetyped** [comment, marked as answer]
Hi, links are required for a lightbox to work.  Without links, there is nothing for visitors to click on and launch the lightbox.

If you are still experiencing an issue after enabling links on the gallery items, please [create a support ticket](https://github.com/archetyped/simple-lightbox/issues/new/choose) so that your issue can be fully evaluated.

Thanks.

**Reply by thaibluesky** [reply]
He writes clearly that all files are linked to mediafile:
 - is linking to mediafile  not a link? 
 I see an a href="correct path to image" and lightbox l is not present in the link and not in the footer - just a css is loaded in the head. II guess cause frontend files are not fully loaded. Are you not using any  js files at all?
